sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -293,35 +293,24 @@ export interface KeyReleasePolicy {
 }
 
 // @public
-export interface KeyRotationLifetimeActions {
-    action?: "Rotate" | "Notify";
-    trigger?: KeyRotationLifetimeActionsTrigger;
-}
-
-// @public
-export interface KeyRotationLifetimeActionsTrigger {
+export interface KeyRotationLifetimeAction {
+    // Warning: (ae-forgotten-export) The symbol "KeyRotationPolicyAction" needs to be exported by the entry point index.d.ts
+    action?: KeyRotationPolicyAction;
```

Comment:
Sorry I didn't catch this earlier, but seems this shouldn't be optional. Could be a bug in the swagger where they didn't mark it as required and should. I could see where either property below wouldn't be, as either is required and swagger doesn't have a great way to express that.

Reply:
Yeah, it's not marked as required. I can follow up